lib/Sema/TypeCheckPropertyWrapper.cpp

```diff
@@ -385,6 +409,16 @@ PropertyWrapperTypeInfoRequest::evaluate(
     }
   }
 
+  result.requireNoEnclosingInstance =
+      enclosingInstanceTypeIsNever(ctx, result.enclosingInstanceWrappedSubscript);
+//  if (requireNoEnclosingInstance) { //  && !valueVar->isStatic()) {
+//    // this means that the property wrapper must be declared on a static property
+//    valueVar->diagnose(
+//        diag::property_wrapper_var_must_be_static, valueVar->getName());
+//    return PropertyWrapperTypeInfo();
+//    result
+//  }
```

Comment:
Why is this commented?

Reply:
Thanks, will remove -- this is a leftover, it was the wrong place to do the diagnosis after all so moved elsewhere